src/components/manifold-plan-table/manifold-plan-table.tsx

```diff
@@ -420,48 +403,58 @@ export class ManifoldPlanTable {
                 />
               </p>
             </div>,
-            Object.values(this.planFeatures[plan.id]).map((feature) => {
-              // fixed feature
-              if (feature && this.productFeatures.fixed[feature.label]) {
-                return <FixedFeature displayValue={(feature as PlanFixedFeature).displayValue} />;
-              }
-
-              // metered feature
-              if (feature && this.productFeatures.metered[feature.label]) {
-                return <MeteredFeature feature={feature as PlanMeteredFeature} />;
-              }
-
-              // configurable
-              if (feature && this.productFeatures.configurable[feature.label]) {
-                const configurableFeature = feature as PlanConfigurableFeature;
-                return this.displayConfigurable({
-                  planID: plan.id,
-                  feature: configurableFeature,
-                });
-              }
-
-              // undefined / disabled feature
-              return (
-                <div class="ManifoldPlanTable__Cell ManifoldPlanTable__Cell--Body">
-                  <span class="ManifoldPlanTable__Cell__Disabled">•</span>
-                </div>
-              );
-            }),
-            <div
-              class="ManifoldPlanTable__Cell ManifoldPlanTable__Cell--Body"
-              data-row-last
-              data-column-last={lastColumn}
+            <form
+              action={this.baseUrl}
+              onSubmit={this.onSubmit(plan.id)}
+              style={{ display: 'contents' }}
             >
-              <a
-                data-testid="cta"
-                class="ManifoldPlanTable__Button"
-                id={`manifold-cta-plan-${plan.id}`}
-                href={this.ctaHref(plan.id)}
-                onClick={this.handleCtaClick(plan.id)}
-              >
-                {this.ctaText}
-              </a>
-            </div>,
+              <input hidden name="planId" value={plan.id} />
```

Comment:
👍

Reply:
Oh wait this should be `type="hidden"` (though I think `hidden` may work the same)